### PR TITLE
Creating a VPC for HTTPS and Trustworthy Email scanning

### PR DESCRIPTION
**Pull request by jsf9k**

### CODE REVIEW

File: terraform_bod/docker_ec2.tf [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# The bastion AMI
```
Comment (dav3r) [inline terraform_bod/docker_ec2.tf]: Comment should be "The docker AMI".  Probably needs to be updated in the main terraform dir too.

**Review by felddy — COMMENTED** [review]
Let's use 1024 as the lower bound for the dynamic / ephemeral port definition.   Not that this jibes  with the RFC... https://tools.ietf.org/html/rfc6335#section-6

File: terraform_bod/vpc.tf [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+
+# Allow egress to the public subnet via ephemeral ports
+resource "aws_network_acl_rule" "private_egress_to_public_via_ephemeral_ports" {
+  network_acl_id = "${aws_network_acl.bod_private_acl.id}"
+  egress = true
+  protocol = "tcp"
+  rule_number = 150
+  rule_action = "allow"
+  cidr_block = "${aws_subnet.bod_public_subnet.cidr_block}"
+  from_port = 1025
```
Comment (felddy) [inline terraform_bod/vpc.tf]: 1024

**Comment by jsf9k** [comment]
@dav3r and @felddy, I believe my latest commits correct the errors you guys pointed out.  Please re-review the pull request.

**Review by dav3r — APPROVED** [review]

**Review by felddy — APPROVED** [review]

**Pull request merged**